### ISSUE TITLE
Templates should be moved away from the source code

### ISSUE THREAD

**Issue by DePierre** [owtf/http-request-translator#29]
The templates that are used to generate the different scripts are in the same files as the python code to generate the scripts.

It makes it hard for one to read the source code and it is also a poor design to keep it that way.

A better design would be to separate the templates from the source code.

**Comment by viyatb** [comment]
Exactly. A separate templates directory with all the templates will be more clean.
To replace the values within a template, you can use certain placeholders for variables like 
`curl -v {{target}} ....` and replace variables in "{{ }}" with the corresponding values in the actual script generator.

**Comment by DePierre** [comment]
@delta24 Thanks for the input. @arunk-s already fixed the problem with a nice solution. Check his fork ;)
The issue will be closed when everything will be merged.

**Comment by viyatb** [comment]
Why not have a plain template instead of using strings like `begin_code = """` etc?
A more simple solution could be something like 

```
#!/usr/bin/env bash
curl --header "{{header}} : {{header_value}}"  | egrep --color ' {{search_string}} |$' -s --request {{method}} {{url}} {{headers}} --include  -x {{proxy}}  --data '{{body}}'
```

and replace the variables directly.

**Comment by DePierre** [comment]
@delta24 It allows better modularity. You could see what I mean by looking as the regex string search option that would, if chosen by the user, add one more piece of script in between.

**Comment by DePierre** [comment]
Fixed by 301a0e09b6596fe101ecbd9581d47329664ad492 thanks to @arunk-s awesome work :)
